.github/workflows/main.yml

```diff
@@ -65,26 +65,26 @@ jobs:
       # environment variable. See the docs for more details:
       # https://prairielearn.readthedocs.io/en/latest/externalGrading/#running-locally-for-development
       run: docker run -td -v /var/run/docker.sock:/var/run/docker.sock -v /tmp/prairielearn:/jobs -e HOST_JOBS_DIR=/tmp/prairielearn --name=test_container prairielearn/prairielearn /bin/bash
+    - name: Run the Python linter
+      run: docker exec test_container /PrairieLearn/docker/lint_python.sh
     - name: Run the JavaScript linter
       run: docker exec test_container /PrairieLearn/docker/lint_js.sh
     - name: Run the HTML linter
       run: docker exec test_container /PrairieLearn/docker/lint_html.sh
-    - name: Run the Python linter
-      run: docker exec test_container /PrairieLearn/docker/lint_python.sh
-    - name: Run the JavaScript typechecker
-      run: docker exec test_container /PrairieLearn/docker/typecheck_js.sh
     - name: Run the Python typechecker
       run: docker exec test_container /PrairieLearn/docker/typecheck_python.sh
+    - name: Run the JavaScript typechecker
+      run: docker exec test_container /PrairieLearn/docker/typecheck_js.sh
+    - name: Run the Python tests
```

Comment:
Moved the Python testing to run before the javascript. Although a bit off-issue, I think this is a good idea since the Python is just backend for the javascript anyway, and it's annoying to have to sometimes wait 10 minutes for javascript tests to run just to have a Python test fail.